test/reuse/util/userSettings/public.key

```diff
@@ -0,0 +1,8 @@
+-----BEGIN RSA PUBLIC KEY-----
```

Comment:
@DIFSRIP | @cvakiitho : do we need to set/overwrite the priv key in the gh-actions

Reply:
no, you should use the default keys. I guess it is not required to add the key here, can you check?